feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/reply/ReplyTextFieldVisibilityUITest.kt

```diff
@@ -0,0 +1,168 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.reply
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.imePadding
```

Comment:
There are many unused imports here, please remove them.
To prevent this in the future, I suggest to do this automatically when commiting by going to Settings > Version Control > Commit > Commit Checks > Optimize Imports

Reply:
Thanks for the hint! I just changed it :)